### PR TITLE
Task #48974 [Admin] Delete RoomView

### PR DESCRIPTION
**Pull request by anhlh-3049**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/48974

## WHAT (optional)
- Destroy RoomView

## Evidence (Screenshot or Video)
Screenshot from 2022-05-20 16-35-34

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: app/controllers/admin/room_views_controller.rb [hunk 1]
```diff
@@ -13,11 +15,24 @@ def create
       flash[:success] = t ".flash_sucsses"
       redirect_to admin_room_views_path
     else
+      flash[:error] = t ".flash_error"
       render :new
     end
   end
 
+  def destroy
+    @room_view.destroy
```
Comment (ngocvt-0484) [inline app/controllers/admin/room_views_controller.rb]: ?

File: app/controllers/admin/room_views_controller.rb [hunk 2]
```diff
@@ -13,11 +15,24 @@ def create
...
+    flash[:success] = t ".flash_sucsses"
+    redirect_to admin_room_views_path
+  end
+
   private
+  def find_room_view
+    @room_view = RoomView.find_by(id: params[:id])
+    flash[:danger] = t ".flash_error"
```
Comment (ngocvt-0484) [inline app/controllers/admin/room_views_controller.rb]: ?

File: app/controllers/admin/room_views_controller.rb [hunk 3]
```diff
@@ -13,11 +15,24 @@ def create
...
 
+  def destroy
+    @room_view.destroy
+    flash[:success] = t ".flash_sucsses"
+    redirect_to admin_room_views_path
+  end
+
   private
+  def find_room_view
+    @room_view = RoomView.find_by(id: params[:id])
```
Comment (ngocvt-0484) [inline app/controllers/admin/room_views_controller.rb]:
```suggestion
    @room_view = RoomView.find_by id: params[:id]
```

**Comment by anhlh-3049** [comment]
ready

**Pull request merged**